docs/dev/engine_overview.rst

```diff
@@ -1,310 +1,317 @@
+   ======================= =========== ========================================================
 
 .. _engine settings:
 
-settings.yml
-------------
-
-======================= =========== =============================================
-argument                type        information
-======================= =========== =============================================
-name                    string      name of search-engine
-engine                  string      name of searx-engine
-                                    (filename without ``.py``)
-enable_http             bool        enable HTTP
-                                    (by default only HTTPS is enabled).
-shortcut                string      shortcut of search-engine
-timeout                 string      specific timeout for search-engine
-display_error_messages  boolean     display error messages on the web UI
-proxies                 dict        set proxies for a specific engine
-                                    (e.g. ``proxies : {http: socks5://proxy:port,
-                                    https: socks5://proxy:port}``)
-======================= =========== =============================================
-
-
-overrides
+Engine ``settings.yml``
+-----------------------
+
+For a more  detailed description, see :ref:`settings engine` in the :ref:`settings.yml`.
+
+.. table:: Common options in the engine setup (``settings.yml``)
+   :width: 100%
+
+   ======================= =========== ===============================================
+   argument                type        information
+   ======================= =========== ===============================================
+   name                    string      name of search-engine
+   engine                  string      name of searx-engine (filename without ``.py``)
+   enable_http             bool        enable HTTP (by default only HTTPS is enabled).
+   shortcut                string      shortcut of search-engine
+   timeout                 string      specific timeout for search-engine
+   display_error_messages  boolean     display error messages on the web UI
+   proxies                 dict        set proxies for a specific engine
+                                       (e.g. ``proxies : {http: socks5://proxy:port,
+                                       https: socks5://proxy:port}``)
+   ======================= =========== ===============================================
+
+.. _engine overrides:
+
+Overrides
 ---------
 
-A few of the options have default values in the engine, but are often
-overwritten by the settings.  If ``None`` is assigned to an option in the engine
-file, it has to be redefined in the settings, otherwise searx will not start
-with that engine.
+.. sidebar:: engine's global names
 
-The naming of overrides is arbitrary.  But the recommended overrides are the
-following:
+   Global names with a leading underline are *private to the engine* and will
+   not be overwritten.
```

Comment:
Technically, names starting with `_` can be overwritten but they can be `None`.

Reply:
Ooops  .. I forgot to reflect your hint into the documentation.  What Do you think, is it better to fix the documentation or is it better to fix the code that overrides _private members_?

Reply:
I think it is better to fix the documentation.